### ISSUE TITLE
ENH Use namedtuples for universe keys

### ISSUE THREAD

**Issue by drewejohnson** [CORE-GATECH-GROUP/serpent-tools#338]
Sometimes when iterating over universes, it can be difficult to determine what index each element of the keys (0, 0, 0., 0) signify. I propose storing these as [`collections.namedtuple`](https://docs.python.org/3/library/collections.html?highlight=namedtuple#collections.namedtuple) to improve readability. This way, one can do the following:
```
for key in resreader.universes:
    univid = key.universe
    time = key.days
    bu = key.bu
    step = key.step
```
rather than having to constantly remember what index of the key corresponds to universe id or burnup.

I propose that these be applied to
- [x] [result reader universes](https://github.com/CORE-GATECH-GROUP/serpent-tools/blob/88b5707ebb49f9fe3d4755999c0659fa6b5ee131/serpentTools/parsers/results.py#L313)
- [x] [branch containers](https://github.com/CORE-GATECH-GROUP/serpent-tools/blob/88b5707ebb49f9fe3d4755999c0659fa6b5ee131/serpentTools/objects/containers.py#L712)
- [x] [branch collector](https://github.com/CORE-GATECH-GROUP/serpent-tools/blob/88b5707ebb49f9fe3d4755999c0659fa6b5ee131/serpentTools/xs.py#L562)

**Comment by drewejohnson** [comment]
Could implement similar logic for [micro cross section dictionaries](https://serpent-tools.readthedocs.io/en/latest/readers/microxs.html#serpentTools.MicroXSReader.xsVal)

**Comment by drewejohnson** [comment]
Closed with #344 and #343
